docs/table/index.rst

```diff
@@ -44,98 +39,93 @@ case, while the full `astropy.table` documentation is available from the
 
   >>> data_rows = [(1, 2.0, 'x'),
   ...              (4, 5.0, 'y'),
   ...              (5, 8.2, 'z')]
-  >>> t = Table(rows=data_rows, names=('a', 'b', 'c'), meta={'name': 'first table'},
-  ...           dtype=('i4', 'f8', 'S1'))
+  >>> t = QTable(rows=data_rows, names=('a', 'b', 'c'), meta={'name': 'first table'},
+  ...            dtype=('i4', 'f8', 'S1'))
 
-There are a few ways to examine the table.  You can get detailed information
-about the table values and column definitions as follows::
+You can now add a |Quantity| column with units in the table as shown below.  This
+stores a native |Quantity| within the table and brings the full power of
+:ref:`astropy-units` to the table.
 
-  >>> t  # doctest: +IGNORE_OUTPUT_3
-  <Table length=3>
-    a      b     c
-  int32 float64 str1
-  ----- ------- ----
-      1     2.0    x
-      4     5.0    y
-      5     8.2    z
+::
 
-You can also assign a unit to the columns. If any column has a unit
-assigned, all units would be shown as follows::
+  >>> t['d'] = [10, 20, 30] * u.m / u.s
 
-  >>> t['b'].unit = 's'
-  >>> t  # doctest: +IGNORE_OUTPUT_3
-  <Table length=3>
-    a      b       c
-           s
-  int32 float64 str1
-  ----- ------- ----
-      1     2.0    x
-      4     5.0    y
-      5     8.2    z
+If you prefer to not use |Quantity| (or the table has no data with units), then
+instead use the |Table| class to create tables.  See :ref:`quantity_and_qtable`
+for details on the differences and use cases.
 
-Finally, you can get summary information about the table as follows::
+There are a few ways to examine the table.  You can get detailed information
+about the table values and column definitions as follows::
 
-  >>> t.info  # doctest: +IGNORE_OUTPUT_3
-  <Table length=3>
-  name  dtype  unit
-  ---- ------- ----
-     a   int32
-     b float64    s
-     c    str1
+  >>> t
+  <QTable length=3>
+    a      b      c       d
+                        m / s
+  int32 float64 bytes1 float64
+  ----- ------- ------ -------
+      1     2.0      x    10.0
+      4     5.0      y    20.0
+      5     8.2      z    30.0
 
-A column with a unit works with and can be easily converted to an
-`~astropy.units.Quantity` object (but see :ref:`quantity_and_qtable` for
-a way to natively use `~astropy.units.Quantity` objects in tables)::
+Finally, you can get summary information about the table as follows::
 
-  >>> t['b'].quantity  # doctest: +FLOAT_CMP
-  <Quantity [2. , 5. , 8.2] s>
-  >>> t['b'].to('min')  # doctest: +FLOAT_CMP
-  <Quantity [0.03333333, 0.08333333, 0.13666667] min>
+  >>> t.info
+  <QTable length=3>
+  name  dtype   unit  class
+  ---- ------- ----- --------
+     a   int32         Column
+     b float64         Column
+     c  bytes1         Column
+     d float64 m / s Quantity
 
-From within the IPython notebook, the table is displayed as a formatted HTML
+From within a Jupyter notebook, the table is displayed as a formatted HTML
```

Comment:
@mwcraig says Lab is the future, so... This does work in Lab, right?
```suggestion
From within a Jupyter notebook or JupyterLab, the table is displayed as a formatted HTML
```

Reply:
I might over simplify things, but Jupyter Lab isn't wraps around Jupyter notebooks? From out low level POV it doesn't matter if a Lab is deployed or users run their notebooks one by one locally?

Reply:
Well, this is a high level documentation, so doesn't hurt to list both?

Reply:
Well there are even more ways to display / interact with a Jupyter notebook, e.g. within PyCharm.  So I'd just leave it at the base one.